packages/baby-jubjub/src/baby-jubjub.ts

```diff
@@ -1,4 +1,4 @@
-import { F1Field, bigintToHexadecimal, bufferToBigint, leBigintToBuffer, leBufferToBigint, scalar } from "@zk-kit/utils"
+import { F1Field, bigIntToHexadecimal, bufferToBigint, leBigintToBuffer, leBufferToBigint, scalar } from "@zk-kit/utils"
```

Comment:
This name change seems inconsistent with other functions in this library.  E.g. leBigintToBuffer doesn't capitalize the `I` in Bigint, so why should this one function do so?

It would be good to standardize on one rule for whether that I should be capitalized.

Reply:
Right! IMO we should use "BigInt" everywhere, as it's the same name JS uses.

Reply:
I've renamed them. I started marking breaking changes in my commits: https://github.com/privacy-scaling-explorations/zk-kit/pull/175/commits/6858f66f6778bb9656ca97a391f8128ff354d0bf.